송원/후위표기식1/index.py

```diff
@@ -0,0 +1,40 @@
+import sys
+sys.stdin = open("input.txt", "rt")
+
+notation = input()
+priority = {
+  '(': 0,
+  '+': 1,
+  '-': 1,
+  '*': 2,
+  '/': 2
+}
+
+answer = ''
+stack = []
+
+for i in notation:
+  if i.isalpha():
+    answer += i
+  else:
+    if len(stack) == 0:
+      stack.append(i)
+    else:
+      if i == ')':
+        while stack[-1] != '(':
+          answer += stack.pop()
+        stack.pop()
+
+      elif i == '(':
+        stack.append(i)
+      else:
+        while priority[stack[-1]] >= priority[i]:
+          answer += stack.pop()
+          if len(stack) == 0:
+            break
```

Comment:
```suggestion
        while priority[stack[-1]] >= priority[i] and len(stack) > 0:
          answer += stack.pop()
```
break문을 넣기 보다는 while 조건문에 추가하는게 좋아보입니다!

Reply:
아하! 그렇네요 ㅋㅋㅋ